README.md

```diff
@@ -929,6 +929,31 @@ config.after(:suite) do
   FileUtils.rm_rf(Dir["#{Rails.root}/spec/test_files/"])
 end
 ```
+
+**Configurating Tests (Factory Girl)**
+
+FactoryGirl setup for the model that's getting the Paperclip attachment:
+
+```ruby
+FactoryGirl.define do
+  factory :asset do
+    attachment { Rack::Test::UploadedFile.new("#path_to_attachment", "attachment_file_type (e.g. image/png)") }
+  end
+end
+```
+
+spec_helper.rb
+
+```ruby
+RSpec.configure do |config|
+  # ...
+  config.after(:suite) do
+     FileUtils.rm_rf(Dir["#path_where_attachments_are_saved"])
+  end
+  #...
+end
+```
```

Comment:
Images will be loaded in `tmp/`, which is always safe to remove, and can stay as a cache. I don't think we'll need this cleanup section.